pkg/virt-api/rest/authorizer.go

```diff
@@ -230,7 +230,7 @@ func isInfoOrHealthEndpoint(req *restful.Request) bool {
 	// /apis/subresources.kubevirt.io/v1alpha3/namespaces/default/virtualmachineinstances/testvmi/console
 	// The /apis/<group>/<version> part of the urls should be accessible without needing authorization
 	pathSplit := strings.Split(httpRequest.URL.Path, "/")
-	if len(pathSplit) <= 4 || (len(pathSplit) > 4 && (pathSplit[4] == "version" || pathSplit[4] == "healthz" || pathSplit[4] == "guestfs")) {
+	if len(pathSplit) <= 4 || (len(pathSplit) > 4 && (pathSplit[4] == "version" || pathSplit[4] == "healthz" || pathSplit[4] == "guestfs" || strings.Contains(pathSplit[4], "profiler"))) {
```

Comment:
@davidvossel This needs improvement

- restrict this to the exact profiler api types and don't use a wild card `strings.Contains`
- unit test this